### DISCUSSION TITLE
Bootstrap 5 with Webpack reduces the JS file size. However, according to the StackBlitz demo, the size is still too large for just one component.

### DISCUSSION THREAD

**Discussion by bhargav-spinx** [twbs/bootstrap#41095] [General]
Want to skip to the end? Download the source code and working demo for this guide from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/webpack). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/webpack?file=index.html) for live editing.

**Comment by jonhubby** [comment]
Even with Webpack, Bootstrap can still be hefty for individual components.
